docs/paper/dev/api/event-api/chat-event.md

```diff
@@ -0,0 +1,156 @@
+---
+slug: /dev/chat-events
+description: How to properly use the new AsyncChatEvent 
+---
+
+# Chat Events
+The chat event has evolved a couple of times over the years. 
+This guide will explain how to properly use the new `AsyncChatEvent` and its `ChatRenderer`.
+The `AsyncChatEvent` is an improved version of the old `AsyncPlayerChatEvent` that allows you to render chat messages individually for each player.
+
+## Understanding the Renderer
+
+Before we can start using the new chat event, we need to understand how the new renderer works.
+The renderer is Paper's way of allowing plugins to modify the chat message before it is sent to the player.
```

Comment:
Paper or Adventure?

Reply:
The `ChatRenderer` class is located in `io.papermc.paper.chat.ChatRenderer` so I would say it's papers way of modifying chat messages.